# max z distance between neighboring points within segments
# double
z_gap: 0.10

# minimal leg width
# double
leg_width_min: 0.10

# maximal leg width
# double
leg_width_max: 0.25

# minimum laser scanner points on leg
# double
min_leg_points: 8

# max distance between both legs [m]
# double
max_leg_distance: 0.5

# target coordinate system for detected humans
# string
target_frame: "map"

#-------------------------------------------------------
# only relevant for accumulators

# distance threshold for recognizing another detection as the same person when using different laser scanners with potentially different publish times [m]
# double
same_detection_radius: 0.8

# minimum speed so that a speed vector can be computed without creating noisy speed estimates, in [m] offset to last measurement
# double
minimum_speed_for_speed_vector: 0.15

# maximum life time of a detection before it becomes deleted from the accumulator, can be updated by re-detections, in [s]
# double
maximum_detection_lifetime: 3.0
